.github/actions/e2e_test/action.yml

```diff
@@ -167,26 +167,24 @@ runs:
       with:
         azure_credentials: ${{ inputs.azureIAMCreateCredentials }}
 
-    - name: Create UUID
-      id: create-uuid
+    - name: Create prefix
+      id: create-prefix
       shell: bash
       run: |
         uuid=$(uuidgen)
-        echo "uuid=${uuid%%-*}" >> $GITHUB_OUTPUT
+        echo "uuid=${uuid%%-*}" | tee -a $GITHUB_OUTPUT
+        echo "prefix=e2e_${{ github.run_id }}_${{ github.run_attempt }}_${{ steps.create-prefix.outputs.uuid }}" | tee -a $GITHUB_OUTPUT
```

Comment:
Since we use the prefix for all CSPs (which I support) - did you check if this runs? I remember all had different name specifications / allowed chars for their resources (e.g. underscores, hyphens, etc.)